source/agora/test/Base.d

```diff
@@ -83,6 +83,48 @@ public class TestNetwork : Network
         std.concurrency.register(address, api.tid());
         return api;
     }
+
+    /// Wait a certain time until the nodes have reached discovery
+    /// If after 10 query attempts they still all haven't discovered => assert
+    /// Params:
+    ///   count = Expected number of nodes
```

Comment:
Too many lines for `///`

Reply:
I guess we are going to use `/** */` everywhere for multiline comments? I'm 50% sure that using multiline `///` comments works too. Anyway I will fix it.